io.github.CoyoteTM.json

```diff
@@ -0,0 +1,93 @@
+{
+    "app-id" : "io.github.CoyoteTM",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "19.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "command" : "coyotetm",
+    "tags" : [
+        "nightly"
+    ],
+    "desktop-file-name-prefix" : "(Nightly) ",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network",
+        "--talk-name=org.gtk.vfs",
+        "--talk-name=org.gtk.vfs.*",
+        "--filesystem=host",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options" : {
+        "cflags" : "-O2 -g",
+        "cxxflags" : "-O2 -g",
+        "env" : {
+            "V" : "1"
+        }
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "/man",
+        "/share/man",
+        "/share/gtk-doc",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "fltk",
+            "buildsystem" : "autotools",
+            "config-opts" : [
+                "--enable-xdbe",
+                "--enable-xft"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/fltk/fltk.git",
+                    "branch" : "branch-1.3"
+                }
+            ]
+        },
+        {
+            "name" : "libdb",
+            "buildsystem" : "simple",
+            "subdir" : "build_unix",
```

Comment:
Hm, any reason not to specify `dist` here and keep buildsystem set to `autotools`?

Reply:
yep, the configure script must be run outside of dist directory.